client/src/utils.ts

```diff
@@ -1,6 +1,41 @@
+import { differenceInMinutes, fromUnixTime, toDate } from 'date-fns';
+
 /**
  * @description Convert micro to stacks.
  * @param amountInMicroStacks - the amount of microStacks to convert
  */
 export const microToStacks = (amountInMicroStacks: string | number) =>
   Number(amountInMicroStacks) / Math.pow(10, 6);
+
+export const numberWithCommas = (x: string | number): string => {
```

Comment:
Nice way to solve it, another idea you might want to have a look at is to use the `Intl.NumberFormat` API, could also be used to solve this :)

Reply:
A much cleaner way to achieve the same thing, thanks 💯